python/ray/data/_internal/arrow_block.py

```diff
@@ -112,7 +112,7 @@ def _table_from_pydict(self, columns: Dict[str, List[Any]]) -> Block:
 
     def _concat_tables(self, tables: List[Block]) -> Block:
```

Comment:
Can we test this handles generated var-shaped tensor? It looks only the underlying ArrowTensorArray was tested.

Reply:
Do you mean adding unit tests for `ArrowBlockBuilder`?

Reply:
Yeah, just let it concat tables seems to be good enough